toolkit/modules/power_model/data_models/views/cogshop/ShopTimeResolution.view.yaml

```diff
@@ -0,0 +1,42 @@
+space: "{{power_ops_models_space}}"
+externalId: ShopTimeResolution
+name: ShopTimeResolution
+description: The time resolution to used within a SHOP simulation.
+filter:
+  and:
+    - hasData:
+        - type: container
+          space: "{{power_ops_models_space}}"
+          externalId: ShopTimeResolution
+    - equals:
+        property:
+          - node
+          - space
+        value: "{{power_ops_instance_space}}"
```

Comment:
**comment**: I think we should allow other spaces for instances (but now I saw that we have the same filter for many other (all?) views, so I guess that should be dealt with separately, if we want it. Is there a good reason to require all PowerOps instances to be in the same space, @nodegard?